src/Sfa.Tl.ResultsAndCertification.Application/Services/AdminDashboardService.cs

```diff
@@ -51,6 +68,36 @@ public async Task<AdminLearnerRecord> GetAdminLearnerRecordAsync(int pathwayId)
 
             return _adminLearnerRecord;
         }
+
+        public async Task<bool> ProcessChangeStartYearAsync(ReviewChangeStartYearRequest request)
+        {
+            var pathway = await _tqRegistrationPathwayRepository.GetManyAsync(p => p.Id == request.PathwayId
```

Comment:
Why do you have to select and order? You should be able to send the **RegistrationPathwayId** which identifies the specific row in the table. I don't know why the property is always being called **PathwayId** when it is actually the **RegistrationPathwayId**.

Reply:
updated the property name